src/components/HelpChat/HelpChat.js

```diff
@@ -40,41 +41,64 @@ class HelpButton extends React.Component {
 		});
 	};
 
-	handleSubmitIssue = () => {
+	toggleLoading = () => {
+		this.setState(({ isLoading }) => ({
+			isLoading: !isLoading,
+		}));
+	};
+
+	handleSubmitIssue = async () => {
 		const { issue, details } = this.state;
 		const { user } = this.props;
+
+		const errorMessage = (
+			<p>
+				We couldn{"'"}t send the request but you can send an e - mail instead to{' '}
+				<a href={`mailto:support@appbase.io?Subject=${issue}&body=${details}`}>
```

Comment:
There should be a call to action button here:

![image](https://user-images.githubusercontent.com/9116042/48847914-b45c7380-edc8-11e8-8787-90c83ca5d733.png)

Reply:
Also, one minor thing - it should be `e-mail` not `e - mail`.